cosmos/operators/eks.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from typing import Any, Sequence
+
+from airflow.exceptions import AirflowException
+from airflow.utils.context import Context
+
+from cosmos.operators.kubernetes import (
+    DbtBuildKubernetesOperator,
+    DbtKubernetesBaseOperator,
+    DbtLSKubernetesOperator,
+    DbtRunKubernetesOperator,
+    DbtRunOperationKubernetesOperator,
+    DbtSeedKubernetesOperator,
+    DbtSnapshotKubernetesOperator,
+    DbtTestKubernetesOperator,
+)
+
+try:
+    from airflow.providers.amazon.aws.hooks.eks import EksHook
+except ImportError as e:
+    raise ImportError("EksOperator is not compatible with airflow < 2.4") from e
```

Comment:
```suggestion
from airflow.providers.amazon.aws.hooks.eks import EksHook
```

Reply:
hi @VolkerSchiewe , we're planning to drop support for Airflow 2.3 in the upcoming Cosmos 1.5 release. The PR is already merged to main. So if you rebase your PR and get the latest changes, the CI would no longer run tests on Airflow 2.3 and then we don't need this Import handling.

Reply:
Very nice, love to remove the 2.3 related code 👍